src/platform/plugins/shared/workflows_management/public/shared/ui/use_formatted_date.ts

```diff
@@ -12,7 +12,7 @@
 import moment from 'moment-timezone';
 import { useCallback } from 'react';
 import { i18n } from '@kbn/i18n';
-import { useUiSetting } from '@kbn/kibana-react-plugin/public';
+import { useKibana } from '../../hooks/use_kibana';
```

Comment:
Use workflows `useKibana` hook so we have a single source of truth for `services`. It simplified tests